commands/generate.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var url = require('url'),
+    path = require('path');
+
+var Q = require('q');
+
+var lib = require('manifoldjs-lib');
+
+var log = lib.log,
+    manifestTools = lib.manifestTools,
+    projectBuilder = lib.projectBuilder,
+    utils = lib.utils;
+
+var build = require('./package');
+
+function getW3cManifest(siteUrl, manifestLocation, callback) {

```

Comment:
Indeed. The previous release relied almost exclusively on callbacks. For this release, we updated many operations to return promises--you'll see most of that work in manifoldjs-lib--but this one slipped under the radar.